### PR TITLE
Updated README.md & CONTRIBUTING.md

### LINKED ISSUE #1095: Update README file
## Description
As a developer,
I need to update README.md,
so that Information available is consistent with the repository.

## Acceptance Criteria
All the following changes will be completed:
* Remove the link provided in Point 1 under `Set up for developers`
* Delete point 5 under `Set up for developers`
* Remove step 1 under `Configure remotes`
* Update command `git merge upstream/master` to `git rebase upstream/develop`
* Update `master` branch to `develop` in the file.
* Add a point under `Set up for developers` which directs to build `Powerup` which is inside the base directory.
* Change the documentation link under `Documentation of PowerUp (Android)` to [link](https://github.com/systers/powerup-android/wiki#overview)
* Correct `Code of Conduct` link provided in `CONTRIBUTING.md`

### Update [Required]
- [ ] README.md

## Definition of Done
- [ ] All of the required items are completed.
- [ ] Approval by 2 mentors.

## Estimation
2 hours 

Issue discussion  : [Slack thread](https://systers-opensource.slack.com/archives/C0C9VK8UE/p1522316456000019)

### PR DESCRIPTION
**Pull request by Sabihashaik**
### Description
Updated README file and CONTRIBUTING.md to include and remove links mentioned in the issue.
Fixes #1095 
### Type of Change:
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)
### Checklist:
**Delete irrelevant options.**

- [x] My PR follows the style guidelines of this project

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by Sabihashaik** [comment]
@geekanamika I have closed the earlier PR. I have removed the conflicts.

**Comment by Sabihashaik** [comment]
@Rimjhim28 I'm trying to squash the commits through `git rebase -i HEAD~2` but it only shows my first commit "Update Read Me" and not the Merge one.

**Comment by Rimjhim28** [comment]
@Sabihashaik try using git amend.

**Comment by Sabihashaik** [comment]
@Rimjhim28  I tried out  `git amend` which squashed them into one commit but in the PR, it shows up as an additional commit (The previous four commits squashed together & the new squashed commit). Not sure how to fix that?

**Comment by geekanamika** [comment]
@Sabihashaik This is happening probably due to the difference between your remote repository & local repo. Both doesn't contain equal number of commits on the particular branch.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
 # Contributing Guidelines
 * You can join our [slack channel](http://systers.io/slack-systers-opensource/).  Each active repo has its own channel to direct questions to (for example #powerup or #portal).  
-* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](http://www.systers.io/code-of-conduct.html).
+* Remember that this is an inclusive community, committed to creating a safe, positive environment.  See the full [Code of Conduct](/code_of_conduct.md).
 ## General Guidelines
-* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](http://www.systers.io).
+* If you’re just getting started work on an issue labeled “First Timers Only” in any project. Additional resources are available on our [website](systers.io).
```
Comment (geekanamika) [inline CONTRIBUTING.md]: Did you missed it ? Website link is fixed by last merged PR & your PR will break it.

File: PowerUp/app/src/main/java/powerup/systers/com/SplashActivity.java [hunk 2]
```diff
@@ -7,28 +7,17 @@
...
 
-    private Handler handler;
-    private Runnable runnable;
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
-        handler = new Handler();
-        runnable = new Runnable() {
+        new Handler().postDelayed(new Runnable() {
```
Comment (geekanamika) [inline PowerUp/app/src/main/java/powerup/systers/com/SplashActivity.java]: Recheck your branch and added files. These changes are unnecessary.

File: PowerUp/app/src/main/java/powerup/systers/com/SplashActivity.java [hunk 3]
```diff
@@ -7,28 +7,17 @@
...
                 finish();
             }
-        };
-        handler.postDelayed(runnable,1000);
-    }
-
-    @Override
-    protected void onDestroy() {
-        super.onDestroy();
-        handler.removeCallbacks(runnable);
```
Comment (Rimjhim28) [inline PowerUp/app/src/main/java/powerup/systers/com/SplashActivity.java]: Since the PR just concerns with changing documentation, affecting parts of code is not desired. Revert these changes.

Comment (Sabihashaik) [reply PowerUp/app/src/main/java/powerup/systers/com/SplashActivity.java]: @Rimjhim28  Yes I noticed this. Initially It did not have any of those changes, but when I was trying to squash the commits, I think I did something wrong because it added those changes too.
